### PR TITLE
fix WP_Theme_JSON_Resolver_Gutenberg not found

### PR DESCRIPTION
**Pull request by RavanH**
#### Changes proposed in this Pull Request:

Only include Customizer files when Gutenberg is active.

Also: move `add_action( 'after_setup_theme', 'blockbase_support', 9 );` outside conditional.

#### Related issue(s):
https://github.com/Automattic/themes/issues/4768

### CODE REVIEW

File: blockbase/functions.php [hunk 1]
```diff
@@ -137,9 +137,11 @@ function blockbase_restore_customizer() {
 /**
  * Customize Global Styles
  */
-require get_template_directory() . '/inc/customizer/wp-customize-colors.php';
-require get_template_directory() . '/inc/customizer/wp-customize-color-palettes.php';
-require get_template_directory() . '/inc/customizer/wp-customize-fonts.php';
+if ( class_exists( 'WP_Theme_JSON_Resolver_Gutenberg' ) ) {
```
Comment (scruffian) [inline blockbase/functions.php:140]: This is a good idea, thanks.

File: blockbase/functions.php [hunk 2]
```diff
@@ -57,8 +57,8 @@ function( $settings ) {
 		);
 
 	}
-	add_action( 'after_setup_theme', 'blockbase_support', 9 );
 endif;
+add_action( 'after_setup_theme', 'blockbase_support', 9 );
```
Comment (scruffian) [inline blockbase/functions.php:61]: I'm not sure we want to do this. The way I see it, if that function is defined elsewhere, its probably by some totally unrelated code, in which case we probably want to leave it to run itself...

Comment (RavanH) [reply blockbase/functions.php:61]: Ah, okay? I assumed it to be a "pluggable" function for child themes, like many other Automattic themes use ;)

Comment (scruffian) [reply blockbase/functions.php:61]: Oh yes, I see what you mean. 👍
